systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectS2IUtils.java

```diff
@@ -5,43 +5,54 @@
+    private static String namespace = kubeClient().getNamespace();
 
     private KafkaConnectS2IUtils() {}
 
     /**
      * Wait until the given Kafka Connect S2I cluster is in desired state.
-     * @param name The name of the Kafka Connect S2I cluster.
+     * @param clusterName The name of the Kafka Connect S2I cluster.
      * @param status desired status value
      */
-    public static void waitForConnectS2IStatus(String name, String status) {
-        LOGGER.info("Waiting for Kafka Connect S2I {} state: {}", name, status);
-        TestUtils.waitFor("Kafka Connect S2I " + name + " state: " + status, Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Constants.TIMEOUT_FOR_RESOURCE_READINESS,
-            () -> KafkaConnectS2IResource.kafkaConnectS2IClient().inNamespace(kubeClient().getNamespace())
-                    .withName(name).get().getStatus().getConditions().get(0).getType().equals(status),
-            () -> StUtils.logCurrentStatus(KafkaConnectS2IResource.kafkaConnectS2IClient().inNamespace(kubeClient().getNamespace()).withName(name).get()));
-        LOGGER.info("Kafka Connect S2I {} is in desired state: {}", name, status);
+    public static void waitForConnectS2IStatus(String clusterName, String status) {
+        LOGGER.info("Waiting for Kafka Connect S2I {} state: {}", clusterName, status);
+        TestUtils.waitFor("Kafka Connect S2I " + clusterName + " state: " + status, Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Constants.TIMEOUT_FOR_RESOURCE_READINESS,
+            () -> kafkaConnectS2IClient().inNamespace(namespace)
+                    .withName(clusterName).get().getStatus().getConditions().get(0).getType().equals(status),
+            () -> StUtils.logCurrentStatus(kafkaConnectS2IClient().inNamespace(namespace).withName(clusterName).get()));
+        LOGGER.info("Kafka Connect S2I {} is in desired state: {}", clusterName, status);
     }
 
-    public static void waitForRebalancingDone(String name) {
-        LOGGER.info("Waiting for Kafka Connect S2I {} to rebalance", name);
-        TestUtils.waitFor("Kafka Connect S2I rebalancing", Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, Constants.TIMEOUT_FOR_RESOURCE_READINESS,
-            () -> {
-                String connect = kubeClient().listPodNames("strimzi.io/kind", "KafkaConnectS2I").get(0);
-                String log = kubeClient().logs(connect);
-                // wait for second occurrence of message about finished rebalancing
-                return (log.length() - log.replace("Finished starting connectors and tasks", "").length()) / "Finished starting connectors and tasks".length() == 2;
-            });
-        LOGGER.info("Kafka Connect S2I {} rebalanced", name);
+    public static void waitForConnectS2IIsReady(String clusterName) {
+        waitForConnectS2IStatus(clusterName, "Ready");
+    }
+
+    public static void waitForConnectS2IIsNotReady(String clusterName) {
```

Comment:
The capitalization of `S2IIs` is a ugly! Maybe it should be `S2iIs`, or maybe just drop the `Is` entirely (I guess for this and all the other `IsReady`/`IsNotReady` methods?

Reply:
Yes, you are right, it looks like Roman number .. I will remove it, thanks :)